Controllers/Controller.cs

```diff
@@ -0,0 +1,62 @@
+using Microsoft.AspNetCore.Mvc;
+using todoDotNet6.TodoRepo;
+
+namespace todoDotNet6.Controllers
+{
+    [ApiController] 
+    [Route("task")]
+    public class Controller : ControllerBase
+    {
+        private readonly ITodoRepository _todoRepository;
+
+        public Controller(ITodoRepository todoRepository)
+        {
+            _todoRepository = todoRepository;
+        }
+
+        // ITodoRepository todoRepository = new TodoRepo();
+        [HttpGet]
+        public ActionResult<Todo> GetTodos ()
```

Comment:
I'm unclear about how this is working when the return type is `Todo` but you're returning a `IEnumerable<Todo>`

Reply:
So I think why the function accepted the IEnumerable<Todo> and not a Todo is because the function accepts a Todo or an ActionResult and I returned the ActionResult. I changed it to accept an IEnumerable<Todo>

Reply:
Fixed my typo